src/jit/CMakeLists.txt

```diff
@@ -225,6 +222,10 @@ if ((CLR_CMAKE_PLATFORM_ARCH_I386 OR CLR_CMAKE_PLATFORM_ARCH_AMD64) AND WIN32)
     # On x86, build RyuJIT/ARM32 cross-compiling altjit.
     # On amd64, build RyuJIT/ARM64 cross-compiling altjit.
     add_subdirectory(protononjit)
+
+    # Build Linux/x86-running-on-Windows altjit or
+    # Linux/AMD64-running-on-Windows altjit..
+    add_subdirectory(linuxnonjit)
```

Comment:
> Would you mind adding flags to the build that we can set to skip the linuxnonjit and protononjit builds?

Funny, I was thinking of asking the same thing. But then I changed my mind because it doesn't really affect me. Usually I just build the clrjit project in VS and copy the dll to the bin dir. Sure, I still need to do a complete build from time to time but for the most part it's just build clrjit/copy/run test.cmd and repeat.